tests/www/eligibility_views/test_forms.py

```diff
@@ -18,17 +17,15 @@ class TestAdministrativeCriteriaForm:
     """
 
     def test_valid_for_prescriber(self):
-        user = PrescriberFactory()
         criterion1 = AdministrativeCriteria.objects.get(pk=13)
         form_data = {f"{AdministrativeCriteriaForm.LEVEL_2_PREFIX}{criterion1.pk}": "true"}
-        form = AdministrativeCriteriaForm(user, siae=None, data=form_data)
+        form = AdministrativeCriteriaForm(False, siae=None, data=form_data)
```

Comment:
```suggestion
        form = AdministrativeCriteriaForm(is_authorized_prescriber=False, siae=None, data=form_data)
```
facilitera la compréhension du test